lib/private.js

```diff
@@ -69,7 +69,7 @@ class Private {
         const path = `/${API_ROOT}/${V2_API}/${resource}`;
         const headers = {
             'Content-Type': 'application/json',
-            'Content-Length': data.length
+            'Content-Length': Buffer.byteLength(data)
```

Comment:
What is the purpose of the Content-Length header on the V2 API? I don't think we need it, any harm in getting rid of the header altogether?

Reply:
Seems like the getProfileIdByEmail() requires the Content-Length header